src/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -1875,6 +1875,37 @@ public void TestLongProcessIsWorking()
             Assert.True(p.HasExited);
         }
 
+        [PlatformSpecific(TestPlatforms.AnyUnix)]
+        [ActiveIssue(37054, TestPlatforms.OSX)]
+        [Fact]
+        public void LongProcessNamesAreSupported()
+        {
+            string commandName = "sleep";
+
+            if (!IsProgramInstalled(commandName))
```

Comment:
Nit:
```C#
string programPath = GetProgramPath(commandName);
if (programPath == null)
{
    return;
}

... // then use programPath instead of GetProgramPath below
```
?